libraries/core_libs/consensus/src/final_chain/final_chain.cpp

```diff
@@ -18,7 +18,7 @@ class FinalChainImpl final : public FinalChain {
   StateAPI state_api_;
 
   // It is not prepared to use more then 1 thread. Examine it if you want to change threads count
-  util::ThreadPool executor_thread_{1};
+  boost::asio::thread_pool executor_thread_{1};
```

Comment:
Not sure if it will a fast change, because we have 13 usings in the code and 3 in the tests. So I created an issue for this
https://github.com/Taraxa-project/taraxa-node/issues/2239